recipes/fontconfig/all/conanfile.py

```diff
@@ -53,7 +53,7 @@ def requirements(self):
 
     def build_requirements(self):
         self.tool_requires("gperf/3.1")
-        self.tool_requires("meson/1.4.0")
+        self.tool_requires("meson/[>=1.4.0 <1.5]")
```

Comment:
```suggestion
        self.tool_requires("meson/[>=1.4.0 <2]")
```

Expanding the version range to match the upper bound as it is in other recipes